frontend/test/components/AnnotationView/AllianceGenomeCard.spec.js

```diff
@@ -0,0 +1,192 @@
+
+    expect(cardBackground.html()).toContain('[background:] involves: 129S4/SvJae * C57BL/6');
+  });
+
+  it('Displays the Experimental Condition section with black text because it has list items', () => {
+    const wrapper = getMountedComponent();
+    const experimentalSection = wrapper.find('[data-test=model-section-condition]');
+
+    expect(experimentalSection.attributes().style).toContain('color: black;');
+  });
+
+  it('Shows a list of conditions under the experimental conditions section', () => {
+    const wrapper = getMountedComponent();
+    const experimentalSectionList = wrapper.findAll('[data-test=model-list-condition]');
+
+    expect(experimentalSectionList.length).to.equal(1);
+  });
+
+  it('Shows the Associated Human Diseases as gray because it has list items', () => {
+    const wrapper = getMountedComponent();
+    const diseasesSection = wrapper.find('[data-test=model-section-disease]');
+
+    expect(diseasesSection.attributes().style).toContain('color: black;');
+  });
+
+  it('Shows the Associated Human Diseases as gray because there are no list items', () => {
+    const wrapper = getMountedComponent({model: fakeEmptyModelProp});
+    const diseasesSection = wrapper.find('[data-test=model-section-disease]');
+
+    expect(diseasesSection.attributes().style).toContain('--rosalution-grey-300');
+  });
+
+  it('Does not display a list as there are no diseases to show', () => {
+    const wrapper = getMountedComponent({model: fakeEmptyModelProp});
+    const diseasesList = wrapper.find('[data-test=model-list-disease]');
+
+    expect(diseasesList.exists()).to.be.false;
+  });
+
+  it('Displays the Associated Phenotypes section with black text because it has list items', () => {
+    const wrapper = getMountedComponent();
+    const phenotypesSection = wrapper.find('[data-test=model-section-phenotype]');
+
+    expect(phenotypesSection.attributes().style).toContain('color: black;');
+  });
+
+  it('Shows the Associated Phenotypes section as gray because there are no items', () => {
+    const wrapper = getMountedComponent({model: fakeEmptyModelProp});
+    const phenotypesSection = wrapper.find('[data-test=model-section-phenotype]');
+
+    expect(phenotypesSection.attributes().style).toContain('--rosalution-grey-300');
+  });
+
+  it('Displays Phenotype lists with frequent term sub-headers ', () => {
+    const wrapper = getMountedComponent({model: fakeModelProp});
+    const phenotypesList = wrapper.findAll('[data-test=model-list-phenotype]');
+
+    const firstTerm = phenotypesList[0].find('span');
+    const firstTermList = phenotypesList[0].findAll('li');
+
+    expect(firstTerm.html()).toContain('abnormal');
+    expect(firstTermList.length).to.equal(5);
+
+    const secondTerm = phenotypesList[1].find('span');
+    const secondTermIcon = phenotypesList[1].find('font-awesome-icon-stub');
+    const secondTermList = phenotypesList[1].findAll('li');
+
+    expect(secondTerm.html()).toContain('decreased');
```

Comment:
The contains statements needs to be updated to use the Chai Assertion library style.  So for these statements like toContain() they ought to be

`expect(secondTerm.html()).to.include('decreased');`

It could also be .to.equal(''); in other sections also.

Trying to be consistent now that we aren't so much in a rush.

Reply:
The `.toContain` has been updated to `to.include`